datalake-api/src/main/java/com/sequenceiq/sdx/api/model/SdxClusterDetailResponse.java

```diff
@@ -18,7 +18,8 @@ public SdxClusterDetailResponse(SdxClusterResponse sdxClusterResponse, StackV4Re
                 sdxClusterResponse.getEnvironmentCrn(), sdxClusterResponse.getStackCrn(),
                 sdxClusterResponse.getClusterShape(), sdxClusterResponse.getCloudStorageBaseLocation(),
                 sdxClusterResponse.getCloudStorageFileSystemType(), sdxClusterResponse.getRuntime(),
-                sdxClusterResponse.getRangerRazEnabled());
+                sdxClusterResponse.getRangerRazEnabled(),
```

Comment:
Yea this doesn't need to be part of the response.  Remove it for the other files.